database.js

```diff
@@ -0,0 +1,51 @@
+var papers = {entry1:{
```

Comment:
we already have a LOT of files in the top-level dir. can we somehow move this into a subdir, maybe under `doc/`? Or put it on the website and reference that version?